Testreglar/134/App/app-1.3.4a.json

```diff
@@ -70,25 +72,55 @@
 		},
 		{
 			"stegnr": "3.3",
-			"spm": "Finnes det en fungerende mekanisme for å endre visningsretningen?",
-			"ht": "<p>Gjør en visuell inspeksjon, og se etter om det finnes en mekanisme som lar brukeren endre visningsretning. Eksempel på en mekanisme kan være en knapp.</p>\n<p>Aktiver mekanismen og sjekk om visningsretningen for innholdet blir endret.</p>",
+			"spm": "Kan alt innhold på appsiden brukes i begge visningsretninger?",
+			"ht": "<p>Gjør visuell inspeksjon og sjekk om alt innhold kan brukes i begge visningsretninger.</p>",
 			"type": "jaNei",
-			"kilde": [
-				"G214"
-			],
+			"kilde": [],
 			"ruting": {
 				"ja": {
 					"type": "avslutt",
 					"fasit": "Ja",
-					"utfall": "Det finnes en mekanisme for å endre visningsretning."
+					"utfall": "Det er mulig å vise og bruke innholdet både i stående og liggende retning."
 				},
 				"nei": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Beskriv element eller funksjonalitet du ikke fikk til å bruke.",
```

Comment:
Registreres kun brudd her? bør vurdere om man skal registrere samsvar? Eller gir ikke det mening her

Reply:
Det gir ikke mening, fordi brudd og samsvar er på sidenivå, men pga vårt behov for data i rapportering registrer vi elementer med brudd.